en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -1,96 +1,81 @@
 
-1. Go to **Services** from the left navigation.
-2. Create a new service.
-    1. **If this is your first service**, 
     
-        1. Enter a name for the service and click **Create**. 
-           
-           ![Create a new service](../assets/img/observability/first-service-creation.png){.cInlineImage-full}
-           
-    2. **Else** (if you already have services created), 
-        
-        1. Click **+ Create** to start creating a new service.
-        2. Enter a name for the service and click **Create**.
-           
-           ![Create a new service](../assets/img/observability/service-creation.png){.cInlineImage-full}
-
-3. From the left-hand menu, click **Observe**.
-4. On the right-hand bottom, click the **Sample Service** icon.
-   
-    ![Sample service popup](../assets/img/observability/sample-service-pop-up.png){.cInlineImage-full}
-   
-5. Click **Try Sample Service**.
-
-    ![Try sample service](../assets/img/observability/try-sample-service.png){.cInlineImage-half}
+## Step 1: Get started with the sample service
+Follow this procedure to try out the sample service:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
+2. Go to the **Observability** card and click **Try Now**. This takes you to the **Observability** page. 
+3. Click **Try Sample**. You are directed to the sample service low-code view where you can see the following:
+    - Low-code diagram
+    - Throughput graph   
+    - Latency graph
+    - Logs
+    - Diagnostic data
+
+Now you are ready to detect anomalies and perform root cause analysis on the sample service.
     
-## Detect anomalies and perform root cause analysis
-You can detect the anomalies of the usage of a service by observing the **Throughput & Latency** graphs. The throughput graph shows the successful and erroneous requests that occurred during a period. The latency graph shows the latencies of each request. You can observe these graphs at a lower granularity by choosing a shorter time range, which gives you a better understanding of the incidents.
-
-### Step 1: Detect the anomalies
+## Step 2: Detect anomalies and perform root cause analysis
 
-#### Analyze the throughput graph
-The throughput graph shows the throughput of requests per hour for a selected timestamp. 
+To detect anomalies of the service, you must trace its executions from the code statement level to the machine it runs in. You can analyze observability data and logs to troubleshoot and identify the root cause for the anomalies you detect. 
 
-With the sample data, you can observe spikes in the throughput graph.  You can find the root cause for this by learning how to analyze the throughput graph.
+### Analyze the throughput graph
+The throughput graph depicts successful and erroneous requests to the service during a selected time interval. You can analyze the spikes in the throughput graph to identify the root cause for requests that result in an error.
```

Comment:
The throughput graph shows the throughput of requests.  Shall we rephrase " successful and erroneous requests" to "throughput"

Reply:
How about "The throughput graph depicts the total throughput of the service (which includes both successful and erroneous requests) during a selected time interval.

Reply:
Addressed this.